### DISCUSSION TITLE
元宇宙大進擊，Facebook、微軟參戰引爆全球話題

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#126] [General]
元宇宙概念最早在1992年的科幻小說《潰雪》提出，是指由現實與虛擬混合而成的數位世界，允許人們透過虛擬環境在不同裝置間移動及溝通。
隨著虛擬實境（VR）及擴增實境（AR）硬體發展漸漸成熟、上網速度加快、電腦運算能力大幅提升，加上科技大廠的背書，元宇宙躍升為今年熱門關鍵字。
大力推動元宇宙的Facebook認為，元宇宙是一個虛擬世界，人們可以和身處不同空間的人一起打造與探索，也可以在其中與朋友交流、工作、娛樂、學習、購物和創作等。不過元宇宙無法由任何一家公司單獨建立，如同網路世界，許多產品可能在未來10到15年後才會問世。
影音平台Yahoo TV總製作耿暄日前受訪指出，未來元宇宙將有3大人才需求，第一種是內容型人才，必須對內容有豐富的理解和想像力；第二種是技術型人才，對技術有深厚底子，能運用新科技進行創作；第三種是整合型人才，需要促成前述兩種完全不同特質的人才進行對話。
https://technews.tw/2021/10/22/metaverse-facebook-microsoft-global/

**Comment by 109302003** [comment]
波音公司也打算利用元宇宙這個虛擬空間打造飛機。

**Comment by cody910120** [comment]
我覺得老高講得還可以，至少老高很會講故事
